### PR TITLE
new file:   Topic Wise Questions with Solutions/Array/minimize_the_ju…

### PR DESCRIPTION
**Pull request by soumik2012**
### Have you read the Contributing Guidelines on Pull Requests

Yes 

### Description

Sort both arrays then iterators for arr1, arr2 then go along both arrays if an element is lower than another, increment iterator in that array, else part if we find equal elements add an element to the result and increment iterators of both arrays

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

This issue related to #296

### CODE REVIEW

**Comment by soumik2012** [comment]
@rcoder23  please review my PR

File: Topic Wise Questions with Solutions/Array/minimize_the_jump.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+Space Complexity : O(1)
+
+INPUT:
+    Enter the elements: 1 3 5 8 9 2 6 7 6 8 9
+
+OUTPUT  
+    Minimum number of jumps to reach end is 3
+
+Output: 3 (1-> 3 -> 8 -> 9)
+Explanation: Jump from 1st element
```
Comment (sayantani11) [inline Topic Wise Questions with Solutions/Array/minimize_the_jump.py:33]: It's a minute change but I actually didn't understand the explanation at once....
Can you please write something like:  The first element in the list is 1 so only one jump is allowed like only if you wish to....The code is fine....But just a request to write a bit about the question in the beginning that's it

Comment (sayantani11) [reply Topic Wise Questions with Solutions/Array/minimize_the_jump.py:33]: Because I am unable to understand the question at one go

**Review by sayantani11 — CHANGES_REQUESTED** [review]

File: Topic Wise Questions with Solutions/Array/minimize_the_jump.py (hunk 1 above)
Comment (soumik2012) [reply Topic Wise Questions with Solutions/Array/minimize_the_jump.py]: Sure @sayantani11 please check my file I modified  as you suggested

**Comment by soumik2012** [comment]
@sayantani11 I have done this modification as you told please check my PR and one more think please add the gssoc'21 and level label in my PR

**Review by sayantani11 — APPROVED** [review]

**Comment by soumik2012** [comment]
@rcoder23 please add the gssoc'21 label on it and level

**Comment by soumik2012** [comment]
@sayantani11 ma'am please add gssoc'21 label and level 3 label at least

**Comment by soumik2012** [comment]
@rcoder23 please i request to please merge my PR

**Review by rcoder23 — CHANGES_REQUESTED** [review]
For Input : 0 2 1 
print message that can't jump

**Comment by soumik2012** [comment]
@rcoder23 i have change file please check and add tag in here this is question of level 2 add tag here please

**Comment by soumik2012** [comment]
If you don't trust me this is medium level of question gfg and leetcode 
Screenshot_20210415_174844
Here!

**Review by rcoder23 — APPROVED** [review]
Good work

**Pull request merged**